### DISCUSSION TITLE
Richiesta urgente

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#784] [Richieste di aiuto] [answered]
Buonasera

In caso di smarrimento delle credenziali admin come si può procedere?
Mi è appena successo.
Grazie in anticipo

**Comment by trinko** [comment, marked as answer]
Puoi fare in questo modo:
- dalla pagina del tuo servizio di hosting, apri PHPMyAdmin per la gestione del database
- nel database, seleziona sulla sinistra la tabella **gs_utente**
- usando la funzione di ricerca (nel menu in alto) puoi fare una ricerca nella tabella per trovare l'utente con username **admin**
- una volta che viene visualizzata la riga che corrisponde all'utente admin, puoi modificare la password, cliccandoci sopra ti permette di inserirne una nuova
- puoi usare la seguente stringa codificata che corrisponde alla password **12345678**:
```
$2y$13$JwJygXTuNXYDUqMIV7B8KuOWOS8QHf3ZT9i.4Xa7QOg1hVMl9im8K
```
- una volta modificata la password puoi entrare con la password **12345678**, ma ricorda di modificare subito la password con una sicura.

Tieni presente che potrebbe non funzionare perché la codifica usata per il tuo utente potrebbe essere diversa da quella della stringa che ho riportato.

Se hai però un altro tuo utente di cui conosci la password, con lo stesso sistema, puoi copiare la password codificata presente nel database di quell'utente su quella di **admin**.

**Reply by Davide031** [reply]
Va bene, grazie.
Ho risolto.

Arrivederci
